docs/en/query_language/system.md

```diff
@@ -1,9 +1,58 @@
 # SYSTEM Queries {#query_language-system}
 
+- [RELOAD DICTIONARIES](#query_language-system-reload-dictionaries)
+- [RELOAD DICTIONARY](#query_language-system-reload-dictionary)
+- [DROP DNS CACHE](#query_language-system-drop-dns-cache)
+- [DROP MARKS CACHE](#query_language-system-drop-marks-cache)
+- [FLUSH LOGS](#query_language-system-flush_logs)
+- [RELOAD CONFIG](#query_language-system-reload-config)
+- [SHUTDOWN](#query_language-system-shutdown)
+- [KILL](#query_language-system-kill)
 - [STOP DISTRIBUTED SENDS](#query_language-system-stop-distributed-sends)
 - [FLUSH DISTRIBUTED](#query_language-system-flush-distributed)
 - [START DISTRIBUTED SENDS](#query_language-system-start-distributed-sends)
 
+## RELOAD DICTIONARIES {#query_language-system-reload-dictionaries}
+
+Reloads all dictionaries that have been successfully downloaded before.
+By default, dictionaries are loaded lazily (see [dictionaries_lazy_load](../operations/server_settings/settings.md#dictionaries-lazy-load)), so instead of being loaded automatically at startup, they are initialized on first access through dictGet function or SELECT to ENGINE = Dictionary. The `SYSTEM RELOAD DICTIONARIES` query reloads such dictionaries (LOADED).
```

Comment:
```suggestion
By default, dictionaries are loaded lazily (see [dictionaries_lazy_load](../operations/server_settings/settings.md#dictionaries-lazy-load)), so instead of being loaded automatically at startup, they are initialized on first access through dictGet function or SELECT from tables with ENGINE = Dictionary. The `SYSTEM RELOAD DICTIONARIES` query reloads such dictionaries (LOADED).
```